### PR TITLE
docs: misc improvements (but mostly related to Rspamd)

### LINKED ISSUE #3857: question: Default enabled services
### 📝 Preliminary Checks

- [X] I tried searching for an existing issue and followed the [debugging docs](https://docker-mailserver.github.io/docker-mailserver/latest/config/debugging/) advice, but still need assistance.

### 👀 What Happened?

Hi, looking into DMS as a replacement for my current mail setup. I noticed in the documentation `ENABLE_AMAVIS` is on by default, and its description says "used for ClamAV & SpamAssassin". However, both ClamAV (`ENABLE_CLAMAV`) and SpamAssassin (`ENABLE_SPAMASSASSIN`) are turned off by default. Does Amavis then still need to be active or could/should it be disabled too?

This also begs the general question how to deal with combination of services, as I could not find this in the documentation. E.g. I've read to disable policyd when enabling rspamd, but what about the other things rspamd can do? Edit: I do see in the DMS rspamd documentation the "very basic configuration" where a bunch of other services are disabled. However, greylisting for example is not being disabled, but something rspamd can do as well (through `RSPAMD_GREYLISTING`).

Finally, is it correct (read it in 1 of the issues) DMS is moving fully to rspamd (enabled by default) in the future? Is rspamd the safest choice for a new setup, to be compatible with future releases?

### 👟 Reproduction Steps

_No response_

### 🐋 DMS Version

v13.3.1

### 💻 Operating System and Architecture

FedoraCoreOS 39 x86_64

### ⚙️ Container configuration files

_No response_

### 📜 Relevant log output

_No response_

### Improvements to this form?

_No response_

### PR DESCRIPTION
**Pull request by georglauterbach**
# Description

Various fixes to our Rspamd docs, CHANGELOG and PR template (wording). GitHub uses an advanced diff that makes the wording changes to the Rspamd docs page easier to review, which is nice.

Fixes #3857 but does even more

Just FYI: I have finally gotten around to bying [LanguageTool](https://languagetool.org) Premium, which I used for my masters thesis too. Many wording changes were pointed out by this tool; for someone on the looks for such a tool, I can highly recommend it :)

## Type of change

- [x] This change is a pure documentation update

### CODE REVIEW

**Review by polarathene — CHANGES_REQUESTED** [review]
LGTM 👍 

Some suggestions, mostly the ref link naming convention and DMS repo links (_use tag instead of master branch_)

---

> Many wording changes were pointed out by this tool; for someone on the looks for such a tool, I can highly recommend it :)

Quite cool, reminds me of Grammarly :)

File: docs/content/config/security/rspamd.md [hunk 1]
```diff
@@ -195,31 +178,36 @@ You can also have comments (the line starts with `#`) and blank lines in `custom
...
-
 ## Examples & Advanced Configuration
 
 ### A Very Basic Configuration
 
-You want to start using Rspamd? Rspamd is disabled by default, so you need to set the following environment variables:
+Do you want to start using Rspamd? Rspamd is disabled by default, so you need to set the following environment variables:
 
-```env
+```ini
```
[the remark this answers is not in the document]
Comment (georglauterbach) [reply docs/content/config/security/rspamd.md:187]: VS Code does not render `env` correctly for me.

**Comment by georglauterbach** [comment]
> LGTM 👍
> 
> Some suggestions, mostly the ref link naming convention and DMS repo links (_use tag instead of master branch_)

👍🏼 

> > Many wording changes were pointed out by this tool; for someone on the looks for such a tool, I can highly recommend it :)
> 
> Quite cool, reminds me of Grammarly :)

Yeah, but for less money and for more languages 🚀 It's funded by the EU directly as well, hence many languages in the EU are present.

**Review by casperklein — APPROVED** [review]

File: docs/content/config/security/rspamd.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (polarathene) [reply docs/content/config/security/rspamd.md:187]: - Reason: https://markdown-all-in-one.github.io/docs/guide/syntax-highlighting-for-fenced-code-blocks.html#in-editor
- Supported languages: https://github.com/microsoft/vscode-markdown-tm-grammar/blob/main/build.js

Fixing that would require a PR to that `build.js` file, or an alternative described in the reason link.

`ini` is not semantically correct for what we're documenting. If it was just the syntax highlighting in the actual docs as an issue, that'd be an easy fix with an alias in `mkdocs.yml`.

---

Do we want to favor `ini` for VSCode to render the syntax? I had a look for ENV extensions on the marketplace that might have helped but came up empty.

**Review by polarathene — APPROVED** [review]

File: docs/content/config/security/rspamd.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (casperklein) [reply docs/content/config/security/rspamd.md:187]: This seems to work quite well --> In VScode user `settings.json`:

```json
	"files.associations": {
		"*.env": "shellscript"
	},
```

[the remark this answers is not in the document]
Comment (georglauterbach) [reply docs/content/config/security/rspamd.md:187]: Agreed, we'll stick to `env` then 👍🏼

**Pull request merged**